Global Tensor 分布式并行策略 (#512)
Oneflow-Inc/oneflow-documentation: en/docs/cookies/global_tensor_distributed.md

> # Using Global Tensor for Distributed Programming: Distributed Parallelism Strategies 

Reviewer: -> Distributed Programming with Global Tensor ?

> # Using Global Tensor for Multi-Device Multi-GPU Programming: Distributed Parallelism Strategies 

Reviewer: > Multi-Device Multi-GPU

"多机多设备编程"，现在感觉太啰嗦了，可以改了 “分布式编程”，这里翻译需要调整为 distributed programming

请帮忙把这两篇文章都改一下：
- 使用 Global Tensor 进行多机多设备编程：基础操作: cookies/global_tensor.md
- 使用 Global Tensor 进行多机多设备编程：分布式并行策略: cookies/global_tensor_distributed.md

### 数据并行
### Data Parallelism

数据并行是将数据进行切分输入不同设备，而每个设备上的模型保持完整和一致。

In data parallelism, we input different data shards into different devices, and each device runs the same whole model to process its given data shard.

OneFlow 特有的 Global Tensor 采用 `placement` 与 `sbp` 结合的方式完成分布。其中 `placement` 表示 Global Tensor 分布的物理设备，`sbp` 表示 Global Tensor 分布的方式（详情可见：[创建 Global Tensor](./global_tensor.md/#global-tensor_2)）。

In OneFlow’s unique Global Tensor, the distribution is implemented via `placement` and `sbp`. `placement` refers to the physical devices that the global tensor is distributed among, and `sbp` refers to the way that the global tensor is distributed. (For more information, please refer to [Create a Global Tensor](https://docs.oneflow.org/en/master/cookies/global_tensor.html))

以两卡并行为例，矩阵乘法案例的数据并行程序如下：

Take two-GPU parallelism as an example, the data parallelism program for the aforementioned matrix multiplication is as follows:

**注意：没有多个 GPU 的读者，可以通过将本文并行示例中的 `placement` 指定为 `type="cpu"`， 实现用 CPU 模拟多设备并行**

> **Note: If you don’t have multiple GPUs, you can designate the `placement` as `type="cpu"` in the third line of the following code, so you can simulate multi-device parallelism with CPUs.**

Author: ```suggestion
**Note: If you don’t have multiple GPUs, you can simulate multi-device parallelism with CPUs via designating the `placement` as `type="cpu"`.**
```

Reviewer 2: @Jiachuann
